docs/main/glossary.md

```diff
@@ -33,6 +33,7 @@ description: This section lists the abbreviations used thorughout the OpenEBS do
 | PV | Persistent Volume |
 | PVC |	Persistent Volume Claim |
 | RBAC | Role-Based Access Control |
+| SSH | Secure Socket Shell |
```

Comment:
Is this already present?
GKE -> Google Kubernetes Engine

Reply:
Yes Vishnu. GKE is already present in the Glossary.